### PR TITLE
Wip

### PR DESCRIPTION
**Pull request by blemia51**

### CODE REVIEW

**Comment by rocambille** [comment]
make a separate PR for sql files please :)

File: bdd/donneesvoyages.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+*** Destinations***
```
Comment (rocambille) [inline bdd/donneesvoyages.md]: remove this file :)

File: bdd/tuktuk_v01.sql [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+-- CREATE TABLE `travels_users` (
+--     `travel_userID` INT NOT NULL AUTO_INCREMENT,
+--     `rate` INT,
+--     `comment` TEXT,
+--     FOREIGN KEY (`IDuser`) REFERENCES `users`(`userID`),
+--     FOREIGN KEY (`IDtravel`) REFERENCES `travels`(`travelID`),
+--     PRIMARY KEY (`travel_userID`)
+    
+-- );
```
Comment (rocambille) [inline bdd/tuktuk_v01.sql]: dead code: to remove :)

**Review by rocambille — CHANGES_REQUESTED** [review]

File: bdd/donneesvoyages.md (hunk 1 above)
Comment (blemia51) [reply bdd/donneesvoyages.md]: Done

File: bdd/tuktuk_v01.sql (hunk 2 above)
Comment (blemia51) [reply bdd/tuktuk_v01.sql]: Done

**Pull request merged**